include/grpcpp/server_impl.h

```diff
@@ -199,6 +199,14 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
 
   grpc_server* server() override { return server_; }
 
+ protected:
+  void SetHealthCheckServiceInterface(
```

Comment:
Since these are simple accessors, suggest calling them `set_health_check_service()` and `health_check_service_disabled()`.

Reply:
Done.